test/middleware_test.rb

```diff
@@ -90,6 +90,19 @@ def test_amzn_trace_id_in_wrong_format
     assert_invalid_headers_response
   end
 
+  def test_redis_5_patching
+    configure_middleware
+
+    redis_config = RedisClient.config(host: "127.0.0.1", port: 10)
+    redis = redis_config.new_pool(timeout: 0.5, size: 1)
+    PrometheusExporter::Instrumentation::MethodProfiler.start
+    redis.call("PING") # => "PONG"
+    redis.call("PING") # => "PONG"
```

Comment:
I think `call_pipelined` has to be tested too.

Reply:
yeah the test is more complicated cause I need an additional middleware, will see what I can do.